docs/guides/governance.md

```diff
@@ -157,7 +157,7 @@ Where `proposal.json` is:
 
 ```json
 {
-  "title": "Community Pool Spend",
+  "title": "Community Pool Spend: My Cool Proposal",
```

Comment:
If we're trying to make it more suggestive, maybe something like "My cool dApp" or "Super important committee" would be better in this case?

Reply:
Sounds good to me